src/index.js

```diff
@@ -0,0 +1,155 @@
+import './style.css';
+import axios from 'axios';
+import itemsCounter from './modules/counter.js';
+import { getMovies, GetLikes } from './modules/request.js';
+
+const main = document.querySelector('main');
+const div = document.querySelector('.movies');
+const comments = document.querySelector('.comments');
+const header = document.querySelector('.shows');
+const body = document.querySelector('body');
+const commentsDiv = document.querySelector('#comments-wrapper');
+let movies = [];
+let commentHTML = '';
+let getCommentsFromAPI = [];
+
+const displayMovie = async () => {
+  movies = await getMovies();
+  const likes = await GetLikes();
+  const counter = itemsCounter(movies);
+  header.innerHTML = 'Shows';
+  header.innerHTML += ` (${counter})`;
+
+  div.innerHTML = '';
+  movies.forEach((movie, index) => {
+    const likeVal = likes[index] !== undefined ? likes[index].likes : 0;
+    div.innerHTML += `
+    <div class="movie" id="${movie.id}">
+    <img src="${movie.image.medium}" alt="${movie.name}">
+    <div class="likes">
+    <p class="like-p">${movie.name}</p>
+    <button class="like-button"><i class="far fa-heart fa-2x" id='heart-${movie.id}'></i></button>
+    </div>
+    <p>${likeVal} likes</p>
+    <button class="comment-button">Comment</button>
+    </div>`;
+  });
+};
+
+const updateLikes = async (ele) => {
+  await axios.post('https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/XMHWey4za3iNnBFD5KUq/likes', { item_id: ele.id });
+  displayMovie();
+};
+
+const popUpHtml = (target) => {
+  comments.innerHTML = `
+    <div id='pop'>
+      <div id='img-div'>
+        <i class='fas fa-times'></i>
+        <img src="${movies[parseInt(target, 10) - 1].image.medium}" alt="${movies[parseInt(target, 10) - 1].name}">
+      </div>
+      <h2>${movies[parseInt(target, 10) - 1].name}</h2>
+      <div id='comment-feature'>
+        <p> Rating: ${movies[parseInt(target, 10) - 1].rating.average}</p>
+        <p> Genres: ${movies[parseInt(target, 10) - 1].genres}</p>
+        <p> Released Date: ${movies[parseInt(target, 10) - 1].premiered}</p>
+        <p> Language: ${movies[parseInt(target, 10) - 1].language}</p>
+      </div>
+    </div>`;
+};
+
+// eslint disable
```

Comment:
- [ ] [OPTIONAL] Kindly refrain from using `eslint disable` anywhere in your code, as that defies the purpose of using linters in the first place.